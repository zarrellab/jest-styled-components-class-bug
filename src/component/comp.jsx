import React from "react";
import styled from "styled-components";

const Container = styled.div`
  color: red;
`;

export const Comp = () => <Container>test</Container>;
