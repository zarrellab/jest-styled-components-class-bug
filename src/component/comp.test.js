import React from "react";
import { render } from "@testing-library/react";
import "jest-styled-components";

import { Comp } from "./comp";

describe("Comp", () => {
  test("should match snapshots", () => {
    const { container } = render(<Comp />);

    expect(container.firstChild).toMatchSnapshot();
    expect(container.firstChild).toMatchSnapshot();
  });
});
