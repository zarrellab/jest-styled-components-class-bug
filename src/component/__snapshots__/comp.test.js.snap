// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comp should match snapshots 1`] = `
.c0 {
  color: red;
}

<div
  class="c0"
>
  test
</div>
`;

exports[`Comp should match snapshots 2`] = `
<div
  class="sc-bdVaJa rOCEJ"
>
  test
</div>
`;
